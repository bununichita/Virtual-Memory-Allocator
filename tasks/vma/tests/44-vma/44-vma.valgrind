==4345== Memcheck, a memory error detector
==4345== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==4345== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==4345== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==4345== Parent PID: 4344
==4345== 
==4345== 
==4345== HEAP SUMMARY:
==4345==     in use at exit: 0 bytes in 0 blocks
==4345==   total heap usage: 13 allocs, 13 frees, 8,434 bytes allocated
==4345== 
==4345== All heap blocks were freed -- no leaks are possible
==4345== 
==4345== For lists of detected and suppressed errors, rerun with: -s
==4345== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
