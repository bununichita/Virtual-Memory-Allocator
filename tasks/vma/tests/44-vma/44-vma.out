Total memory: C8 bytes
Free memory: 180 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 2

Block 1 begin
Zone: A - 1E
Miniblock 1:	A	-	14	| RW-
Miniblock 2:	14	-	1E	| ---
Block 1 end
2
10
20
